browser/ui/webui/brave_new_tab_message_handler.cc

```diff
@@ -38,6 +43,11 @@ base::DictionaryValue GetStatsDictionary(PrefService* prefs) {
   stats_data.SetInteger(
     "fingerprintingBlockedStat",
     prefs->GetUint64(kFingerprintingBlocked));
+#if BUILDFLAG(ENABLE_BRAVE_PERF_PREDICTOR)
+  stats_data.SetInteger(
+    "bandwidthSavedStat",
```

Comment:
clang-format

Reply:
that just intends the lines differently than the rest of the function, but sure...